### ISSUE TITLE
 "Author declined" e-mail notification

### ISSUE THREAD

**Issue by ceci** [agile-alliance-brazil/submissions#103]
**As a...** track organizer
**I want...** to receive an e-mail when an approved author declines the invitation
**In order to...** improve the chances for it to be replaced with another session by providing the newly accepted author with more time to plan his trip.

**Comment by ipepe** [comment]
@hugocorbucci isn't that implemented here: https://github.com/agile-alliance-brazil/submissions/blob/master/app/mailers/email_notifications.rb#L73 ?

**Comment by hugocorbucci** [comment]
Nope. These are the emails that are sent when conference organizers enter the proposal outcome which is, from the organizing committee's point of view, which sessions they would like to have at the conference. Authors have then to accept (or reject) the committee's invitation to speak. When authors reject that invitation, we would like for the organizers to receive an email informing  that the specified proposal will not make it into the program and, hence, they need to either find a replacement or handle that rejection some other way.
Does that make sense?

So, in short, this issue is about triggering an email when authors click on the withdraw link (https://github.com/agile-alliance-brazil/submissions/blob/master/app/views/email_notifications/notification_of_accept.en.html.haml#L28) and the session changes state from pending_confirmation to rejected in the DB. Makes sense now @ipepe?

**Comment by ipepe** [comment]
Definetly more than before, thanks.

**Comment by gmotta** [comment]
@hugocorbucci 
How are we dealing with this nowadays?
When we export the data, we just ignore sessions that have been rejected?
Or, can we actually see this information on any view of the system?

**Comment by hugocorbucci** [comment]
Proposals. that get declined by the author look just like proposals who have been rejected by the committee. So you can find out that an author declined but it's just hard. This would help track leads/reviewing team know they have to pull a new proposal from the backup.

**Comment by hugocorbucci** [comment]
Done in https://github.com/agile-alliance-brazil/submissions/commit/28135499c524ac834f2648c2b3d54642e60c8265
